frame/contracts/src/lib.rs

```diff
@@ -955,9 +957,70 @@ pub mod pallet {
 		StorageValue<_, BoundedVec<DeletedContract, T::DeletionQueueDepth>, ValueQuery>;
 }
 
+/// The type of origins supported by the contracts pallet.
+#[derive(Clone)]
+pub enum ContractOrigin<T: Config> {
```

Comment:
Prefix is redundant as we are already inside the contracts module.

```suggestion
pub enum Origin<T: Config> {
```

Reply:
we don't have `ContractOrigin` anymore, it has been replaced by `Caller`